appium 2.7.0 runtime failure

Trying to install using node 22, but failed to run, error is below:

```
$ appium --version
node:internal/modules/cjs/loader:1189
  throw err;
  ^

Error: Cannot find module '@appium/logger'
Require stack:
- /Users/rui/.nvm/versions/node/v22.2.0/lib/node_modules/appium/build/lib/logsink.js
- /Users/rui/.nvm/versions/node/v22.2.0/lib/node_modules/appium/build/lib/main.js
- /Users/rui/.nvm/versions/node/v22.2.0/lib/node_modules/appium/index.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1186:15)
    at Module._load (node:internal/modules/cjs/loader:1012:27)
    at Module.require (node:internal/modules/cjs/loader:1271:19)
    at require (node:internal/modules/helpers:123:16)
    at Object.<anonymous> (/Users/rui/.nvm/versions/node/v22.2.0/lib/node_modules/appium/build/lib/logsink.js:7:34)
    at Module._compile (node:internal/modules/cjs/loader:1434:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1518:10)
    at Module.load (node:internal/modules/cjs/loader:1249:32)
    at Module._load (node:internal/modules/cjs/loader:1065:12)
    at Module.require (node:internal/modules/cjs/loader:1271:19)
    at require (node:internal/modules/helpers:123:16)
    at Object.<anonymous> (/Users/rui/.nvm/versions/node/v22.2.0/lib/node_modules/appium/build/lib/main.js:9:19)
    at Module._compile (node:internal/modules/cjs/loader:1434:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1518:10)
    at Module.load (node:internal/modules/cjs/loader:1249:32)
    at Module._load (node:internal/modules/cjs/loader:1065:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/rui/.nvm/versions/node/v22.2.0/lib/node_modules/appium/build/lib/logsink.js',
    '/Users/rui/.nvm/versions/node/v22.2.0/lib/node_modules/appium/build/lib/main.js',
    '/Users/rui/.nvm/versions/node/v22.2.0/lib/node_modules/appium/index.js'
  ]
}

Node.js v22.2.0
```

relates to https://github.com/Homebrew/homebrew-core/pull/173894

--- a/packages/appium/package.json
+++ b/packages/appium/package.json
@@ -63,6 +63,7 @@
     "@appium/base-driver": "^9.8.0",
     "@appium/base-plugin": "^2.2.33",
     "@appium/docutils": "^1.0.9",
+    "@appium/logger": "^1.1.0",
     "@appium/schema": "~0.5.0",
     "@appium/support": "^4.3.0",
     "@appium/types": "^0.19.0",

--- a/packages/support/package.json
+++ b/packages/support/package.json
@@ -41,6 +41,7 @@
     "test:unit": "mocha \"./test/unit/**/*.spec.js\""
   },
   "dependencies": {
+    "@appium/logger": "^1.1.0",
     "@appium/tsconfig": "^0.3.3",
     "@appium/types": "^0.19.0",
     "@colors/colors": "1.6.0",

--- a/packages/types/package.json
+++ b/packages/types/package.json
@@ -38,6 +38,7 @@
     "test:types": "tsd"
   },
   "dependencies": {
+    "@appium/logger": "^1.1.0",
     "@appium/schema": "^0.5.0",
     "@appium/tsconfig": "^0.3.3",
     "@types/express": "4.17.21",

